### DISCUSSION TITLE
Secret variavel

### DISCUSSION THREAD

**Discussion by RoqueSPP** [community/community#59764] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

bom dia! 
tenho um projeto robotfrmework e não estou sabendo como posso colocar a variavel "secret" do git actions nos testes.
alguem pode me orientar?

**Comment by DSDmark** [comment, marked as answer]
Hi @RoqueSPP,

First Go to your **repo** in which you want to add **secret**. After that, click on **repo settings** like this:

![image](https://github.com/community/community/assets/85744816/1a1fbfb8-a6c5-4f0f-a5ac-16092351d680)

After open the repo setting. It looks like this:

![image](https://github.com/community/community/assets/85744816/562993e4-824d-498d-8009-dd8f76c8377e)

Now in **secret and variable** section click on **action** option then add your secret by clicking on **new repository secret**.

Now you are done, Now you can use these **secret** in your GitHub action like this:

![image](https://github.com/community/community/assets/85744816/ec0af273-d002-4ee1-a419-3a6771d9d2a1)

And also,

> most of time this above setup work. but, it is the basic implementation of how you can add secret in github action. but, it is alos depent upon use case. So, i suggest you to at least read offical docs for clearty: [**variables**](https://docs.github.com/en/actions/learn-github-actions/variables) and [**encrypted-secrets**](https://docs.github.com/en/actions/security-guides/encrypted-secrets)

**Reply by RoqueSPP** [reply]
Essa parte eu fiz amigo!

 O que não estou sabendo fazer é colocar nos testes essa variável!
 Chamar ela

**Reply by RoqueSPP** [reply]
https://github.com/RoqueSPP/TypeFormRobotframework

**Reply by DSDmark** [reply]
@RoqueSPP, try to change your action `CYPRESS_RECORD_KEY: ${{ secrets.TYPEFORM_ACCESS_TOKEN }}` into this `CYPRESS_RECORD_KEY: ${secrets.TYPEFORM_ACCESS_TOKEN)}` everywhere you write syntax like that.
